pom.xml

```diff
@@ -309,6 +309,50 @@
       <properties>
         <checkstyle.severity>error</checkstyle.severity>
       </properties>
+
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
What are the implications of always running the tests with jacoco? Why don't we enable the coverage collection on demand?

Reply:
Sorry, I didn't notice, that's good then.